readme.md

```diff
@@ -137,8 +137,14 @@ To have commands colorized as seen in the screenshot, install [zsh-syntax-highli
 
 ### [oh-my-zsh](https://github.com/robbyrussell/oh-my-zsh)
 
-1. Symlink (or copy) `pure.zsh` to `~/.oh-my-zsh/custom/themes/pure.zsh-theme`.
-2. Set `ZSH_THEME="pure"` in your `.zshrc` file.
+1. Symlink (or copy) `pure.zsh` to `~/.oh-my-zsh/custom/sindresorhus-pure.zsh-theme`.
+2. Symlink (or copy) `async.zsh` to `~/.oh-my-zsh/custom/async.zsh`.
+3. Set `ZSH_THEME="sindresorhus-pure"` in your `.zshrc` file.
```

Comment:
Oh, didn't know you had started this. Good effort!

Looks like it isn't PR'd yet, but if it gets merged then this definitely is not necessary. I'll change it back to just `pure` for now. Let's revisit if the oh-my-zsh version never gets renamed.